Nosach/web-quiz-project/src/main/java/servlet/RegisterServlet.java

```diff
@@ -21,15 +25,16 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws S
         String login = req.getParameter("login");
         String pass = req.getParameter("pass");
         String email = req.getParameter("email");
+        String name = req.getParameter("name");
 
-        try (FileWriter fw = new FileWriter(getServletContext().getRealPath("/resources/properties/users.properties"), true)) {
 
-            fw.write(login + "#" + pass + "#" + email + "\n");
-            fw.flush();
-        }
+        UserService us = new UserService();
+        PasswordEncoder encoder = new BCryptPasswordEncoder();
```

Comment:
Consider to pass encoding logic to service, servlet is not right place for this, on my point of view